services/todo-legacy/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Application, Server } from '@loopback/core';
+import { setBindings } from './util/setup';
+
+class TodoApplication extends Application {
+  private _startTime: Date;
+
+  constructor() {
+    super();
+    const app = this;
+    setBindings(app);
+  }
+
+  async start() {
+    this._startTime = new Date();
+    const server = new Server(this, { port: process.env.PORT || 3000 });
```

Comment:
Ah, good catch. I created an issue to keep track of this task, see https://github.com/strongloop/loopback-next/issues/412